src/reporter/coverage-report.xsl

```diff
@@ -164,19 +176,35 @@
                (?:[^\]]|\][^\]]|\]\][^>])*   <!-- ?: the content of the CDATA section -->
              \]\]>)
             |
-            (&lt;/                           <!-- 6: a close tag -->
-               ([^>]+)                       <!-- 7: the name of the element being closed -->
+            (&lt;!DOCTYPE                    <!-- 6: a DOCTYPE declaration -->
```

Comment:
Seems like `$group-doctype` is never used. If so, then this group 6 could be a non-capturing group, that is:

```suggestion
            (?:&lt;!DOCTYPE                  <!-- ?: a DOCTYPE declaration -->
```

Reply:
This will still require renumbering of the remaining; will do so.